### DISCUSSION TITLE
prisma doesn't work on apple M1

### DISCUSSION THREAD

**Discussion by sathwik77** [prisma/prisma#21932] [Q&A]
### Bug description

After installing Prisma and running
`npx prisma init`
nothing happens and the process never exists and does not show any logs

### How to reproduce

1. Open any nodejs/bun project in **apple M1** 
2. Run `npm install prisma` or `bun add prisma`
3. Run `npx prsima init` or `bunx prisma init`

### Expected behavior

Should create schema.prisma schema

### Prisma information

prisma version 5.5.2

### Environment & setup

- OS: Apple M1 macOS 13.1
- Database: postgres
- Node.js version: 20.9.0

### Prisma Version

```
5.5.2
```

**Comment by ludralph** [comment]
Hi @sathwik77 👋 

Thank you for raising this question.

Can you please share the content of your package.json file? Do you have the same experience when you use a lower version of Prisma, say for example Prisma version 5.4.2. What happens when you use a lower version of Node.js? Do you experience the same issue?

**Reply by musmanalibaloch** [reply]
I did this as I was facing the issue but it **did not work**

**Reply by ludralph** [reply]
Hi @musmanalibaloch 

Can you try running the command with debug logging enabled? Please share the output 
```
DEBUG="*" npx prisma init
```
 Please share the output once it execute.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
